src/database.ts

```diff
@@ -69,7 +69,17 @@ async function exportAsync(database, path) {
   }
 }
 
+type DatabaseOptions = {
+  version: number,
+  path: string,
+  onUpgrade: (...args: any[]) => any,
+  onDowngrade: (...args: any[]) => any
+};
```

Comment:
```suggestion
interface DatabaseOptions {
  version: number,
  path: string,
  onUpgrade: (...args: any[]) => any,
  onDowngrade: (...args: any[]) => any
}
```

Prefer `interface` when possible.

Reply:
Done :)